pkg/model/file/marker.go

```diff
@@ -24,9 +24,9 @@ import (
 const prefix = "+kubebuilder:scaffold:"
 
 var commentsByExt = map[string]string{
-	// TODO(v3): machine-readable comments should not have spaces by Go convention. However,
-	//  this is a backwards incompatible change, and thus should be done for next project version.
-	".go":   "// ",
+	// it is used just to gen the marker annotations (//+)
+	".go": "//",
+	// It is used just to gen the marker commanets in the yaml files (# +)
 	".yaml": "# ",
```

Comment:
I would re-write the comments inline to keep 1 line per file type and including a longer example:
```suggestion
	".go":   "//", // Marker annotations for go files should not have spaces by go convention: //+kubebuilder:*
	".yaml": "# ", // Marker annotations for yaml files are more readable with spaces: # +kubebuilder:*
```